README.md

```diff
@@ -20,7 +20,7 @@ To get a local copy up and running follw these simple example steps:
     - Clone the repo with git clone using the link below
     - git clone hhttps://github.com/ogiogiovictor/bookstore
     - $ Once the report has been cloned change the project directory by cd into the directory
```

Comment:
- [ ] Also, it would be better if we make use of the API given instead of mocking the data and displaying it as we have currently. Kindly go through the project descriptions again and rectify this